src/app/admin-page/student-list/student-list.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { combineLatest, Observable } from 'rxjs';
+import { filter, take, tap } from 'rxjs/operators';
+import { StudentService } from '../../service/student.service';
+import { Student } from '../../shared/student';
+import { loadStudents } from '../../store/students/students.actions';
+import { selectHasStudentLoadFailed, selectIsStudentsLoaded, selectIsStudentsLoading, selectStudents, selectStudentsError } from '../../store/students/students.selectors';
+
+@Component({
+  selector: 'app-student-list',
+  templateUrl: './student-list.component.html',
+  styleUrls: ['./student-list.component.scss']
+})
+export class StudentListComponent implements OnInit {
+  students$: Observable<Student[]> | undefined;
+
+  isStudentsLoading$: Observable<boolean> =  this.store.select(selectIsStudentsLoading);
+  isStudentsLoaded$: Observable<boolean> = this.store.select(selectIsStudentsLoaded);
+  hasLoadFailed$: Observable<boolean> = this.store.select(selectHasStudentLoadFailed);
+  error$: Observable<Error | null> = this.store.select(selectStudentsError);
+
+  constructor(private store: Store) { }
+
+  ngOnInit(): void {
+
```

Comment:
```suggestion
this.isStudentsLoading$ = this.store.select(selectIsStudentsLoading);
```

Instantiation should be moved here :)